source/_partials/content/https-requirements.html

```diff
@@ -1,10 +1,8 @@
 <h3>Requirements for Automated Certificate Renewal</h3>
 <ul>
-  <li>All A/AAAA records must point to a server that serves Pantheon's acme-challenge response (e.g., <code>http://23.185.0.1/.well-known/acme-challenge/[variableDataHere]</code>).</li>
-  <li>Authority Name Servers must serve mixed-case lookups.</li>
-  <li>Authority Name Servers must not fail CAA lookups.</li>
+  <li>All A/AAAA/CNAME/DNAME DNS records must point to Pantheon's servers so Let's Encrypt can verify domain ownership. MX/NS and similar records can point to other addresses.</li>
```

Comment:
Let's remove the line about MX/NS. They *can't* point to Pantheon if you want things to work, so I think it just confuses things.